src/v/redpanda/request_auth.h

```diff
@@ -68,7 +68,7 @@ class request_auth_result {
      */
     void pass();
 
-    ss::sstring const& get_username() { return _username; }
```

Comment:
Can this be made const in the previous commit?

Reply:
Done